main.py

```diff
@@ -40,6 +40,7 @@ async def on_ready():
 bot.load_extension("cogs.uhoh")
 bot.load_extension("cogs.arii")
 bot.load_extension("cogs.praying")
+bot.load_extension("cogs.dejv")
```

Comment:
would be better to load all cogs with loop instead of adding single lines
e.g.

```python
for name in os.listdir("./cogs"):
    filename = f"./cogs/{name}"
    modulename = f"cogs.{name}"

    if isfile(filename) and filename.endswith(".py"):
        bot.load_extension(modulename[:-3])
```